casanovo/denovo/model_runner.py

```
@@ -185,20 +186,28 @@ def train(
+            The path to the annotation file paths.
         """
         self.initialize_trainer(train=True)
         self.initialize_tokenizer()
         self.initialize_model(train=True)
 
+        # Need to pass in here? idk yet tbh
         train_paths = self._get_input_paths(train_peak_path, True, "train")
         valid_paths = self._get_input_paths(valid_peak_path, True, "valid")
-        self.initialize_data_module(train_paths, valid_paths)
+        annotation_paths = self._get_input_paths(
+            annotation_path, True, "annotations"
```

Ruff (0.15.12)

[warning] Boolean positional value in function call

(FBT003)